models/pg_repo.go

```diff
@@ -0,0 +1,115 @@
+package models
+
+import (
+	"context"
+	"database/sql"
+	"log"
+	"time"
+
+	"github.com/renecouto/logu/psql"
+)
+
+var ctx = context.Background()
+
+func convTaskFromPg(s psql.Task) Task {
```

Comment:
maybe use a single model for front and backend? what problems might come up from that?